### PR TITLE
add readthedocs integration

### PR DESCRIPTION
**Pull request by matthiasdiener**
Here is how it looks like: https://mirgecom.readthedocs.io/en/rtd/

Fixes https://github.com/illinois-ceesd/planning/issues/62

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -61,7 +61,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```
Comment (inducer) [inline doc/conf.py:64]: Is this required?

Comment (matthiasdiener) [reply doc/conf.py:64]: No, but doesn't it look nicer? sphinx_rtd_theme is the "standard" RTD theme, alabaster is here: https://alabaster.readthedocs.io/en/latest/

Comment (inducer) [reply doc/conf.py:64]: I kind of like alabaster better. :)

Comment (inducer) [reply doc/conf.py:64]: I've added that to the agenda for Friday. :P

Comment (matthiasdiener) [reply doc/conf.py:64]: > I've added that to the agenda for Friday. :P

:cold sweat:

I'll set it back to alabaster 😃

**Comment by inducer** [comment]
I didn't see a strong reason not to, so I just went and merged this. Shout if that was a bad idea.

**Pull request merged**